src/schrodinger/sketcher/molviewer/atom_display_settings.cpp

```diff
@@ -17,6 +17,10 @@ void AtomDisplaySettings::setColorScheme(const ColorScheme& scheme,
 {
     // update the annotation color
     AbstractAtomOrBondDisplaySettings::setColorScheme(scheme, carbon_color);
+    m_valence_error_area_color =
+        (scheme == ColorScheme::DARK_MODE || scheme == ColorScheme::WHITE_BLACK)
```

Comment:
I'm a little worried that we're going to add a new dark mode color scheme at some point and forget to update some of the dark mode checks.  We could add something like add
```
const std::unordered_set<ColorScheme> DARK_COLOR_SCHEMES = {
    ColorScheme::DARK_MODE, ColorScheme::WHITE_BLACK};
```
to image_generation.h and then change this (and `SketcherModel::hasDarkColorScheme`) to `DARK_COLOR_SCHEME.contains(scheme)`.  Does that seem reasonable?

Reply:
yep, thanks for adding it